pkg/operation/botanist/etcd.go

```diff
@@ -167,9 +173,9 @@ func (b *Botanist) ScaleETCDToZero(ctx context.Context) error {
 	return b.scaleETCD(ctx, 0)
 }
 
-// ScaleETCDToOne scales ETCD main and events replicas to one.
-func (b *Botanist) ScaleETCDToOne(ctx context.Context) error {
-	return b.scaleETCD(ctx, 1)
+// ScaleUpETCD scales ETCD main and events replicas to one.
```

Comment:
Do we also run 3 replicas for etcd-events? Or is it 3 replicas for etcd-main and only 1 for etcd-events?

Reply:
For now we also scale up `etcd-events` to 3 replicas. I run some tests w/o `etcd-events` running in the control-plane. Even though the Kube-Apiserver keeps running, the cluster itself enters a "low operation" mode, e.g. the `Kube-Controller-Manager` fails executing any task as soon as any even creation is involved.
I can think of optimizations in the future to save resources for this "less important" etcd cluster but that is still tbd and probably exceeds the scope of this PR.

Reply:
Existing KAPIs keep running if additional etcds (like events) go down, however new KAPIs do not start up if not all etcds are available.

Reply:
Oh yes, that's another point, thank you!